app/filters/wpcc/session_expirer.rb

```diff
@@ -1,6 +1,7 @@
 module Wpcc
   class SessionExpirer < ::Wpcc::BaseFilter
-    WPCC_SESSION_EXPIRY_LIMIT = 30.minutes
+    WPCC_SESSION_EXPIRY_LIMIT =
+      ::Wpcc::ConfigLoader.load('session')['expiry_limit'].minutes
```

Comment:
Could we, possibly, load the other constants through the `ConfigLoader` too or will it create too much clutter?

Reply:
All the places that were using yaml file are using the ConfigLoader. Which constant on this file do you expect?

Reply:
Referring to these constants which were not previously stored in a YAML file. 

https://github.com/moneyadviceservice/wpcc/blob/d1915fb0a60cb11ae865702ab6a883b8e1d57072/app/models/wpcc/your_details_form.rb#L9-L12

Reply:
The salary frequencies constant I am already using an yaml on this PR.
The others as gender and age I don't think it is necessary to use a yaml because we are using only for validation and nothing extra.

Reply:
Is it sufficient to have only one test for the config loader? 

```
file: spec/models/your_details_form_spec.rb

   8   describe 'load configuration' do
   9     it 'returns salary frequencies' do
  10       expect(Wpcc::YourDetailsForm::SALARY_FREQUENCIES).to eq(
  11         %w[year month fourweeks week]
  12       )
  13     end
  14   end
```

Places where they appear.
```
Seach results: ConfigLoader
   1 app/filters/wpcc/session_expirer.rb|4 col 15| ::Wpcc::ConfigLoader.load('session')['expiry_limit'].minutes
   2 app/models/wpcc/salary_frequency_converter.rb|3 col 27| CONVERSIONS = ::Wpcc::ConfigLoader.load('salary_frequency_conversions')
   3 app/models/wpcc/your_details_form.rb|12 col 15| ::Wpcc::ConfigLoader.load('salary_threshold').keys.map(&:to_s)
   4 app/models/wpcc/period_contribution_calculator.rb|13 col 15| ::Wpcc::ConfigLoader.load('salary_frequency_conversions')
   5 app/models/wpcc/salary_message.rb|7 col 15| ::Wpcc::ConfigLoader.load('salary_frequency_conversions')
   6 app/models/wpcc/contribution_calculator.rb|3 col 26| THRESHOLDS = ::Wpcc::ConfigLoader.load('earning_thresholds')
   7 app/models/wpcc/period_filter.rb|6 col 23| PERIODS = ::Wpcc::ConfigLoader.load('periods_legal_percents')
   .
   .
   9 app/validators/wpcc/salary_threshold_validator.rb|4 col 15| ::Wpcc::ConfigLoader.load('salary_threshold').stringify_keys
```

My question is, 'is it enough to test the config loader for only the _SalaryFrequencies`?

Reply:
I understand your points, but remove the config loader logic inside the load method, and see what happens when you run the tests (rspec or cucumber).